.github/workflows/packaging.yml

```diff
@@ -3,6 +3,19 @@ name: packaging
 on: [push, pull_request]
 
 jobs:
+  version-check:
+    # Skip pull request jobs when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Run action
```

Comment:
"Check module version" sounds more useful

Reply:
Done, now `name:` is `Check module version`